sklearn/tree/tests/test_tree.py

```diff
@@ -277,10 +291,11 @@ def test_diabetes_overfit(name, Tree, criterion):
 @skip_if_32bit
 @pytest.mark.parametrize("name, Tree", REG_TREES.items())
 @pytest.mark.parametrize(
-    "criterion, max_depth",
-    [("mse", 15), ("mae", 20), ("friedman_mse", 15)]
+    "criterion, max_depth, max_loss",
```

Comment:
I would be in favour to pass the metric in the parametrization as well.
It avoids the if/else condition within the test and will be explicit in the name while executing the test